doc/services/pm/device.rst

```diff
@@ -10,37 +10,73 @@ setting :kconfig:option:`CONFIG_PM_DEVICE` to ``y``. When this option is
 selected, device drivers implementing power management will be able to take
 advantage of the device power management subsystem.
 
-Zephyr supports two types of device power management:
+Zephyr supports two methods of device power management:
 
  - :ref:`Device Runtime Power Management <pm-device-runtime-pm>`
- - :ref:`System Power Management <pm-device-system-pm>`
+ - :ref:`System Managed Device Power Management <pm-device-system-pm>`
 
 .. _pm-device-runtime-pm:
 
 Device Runtime Power Management
 *******************************
 
-In this method, the application or any component that deals with devices directly
-and has the best knowledge of their use, performs the device power management. This
-saves power if some devices that are not in use can be turned off or put
-in power saving mode. This method allows saving power even when the CPU is
-active. The components that use the devices need to be power aware and should
-be able to make decisions related to managing device power.
-
-When using this type of device power management, the kernel can change CPU
-power states quickly when :c:func:`pm_system_suspend()` gets called. This is
-because it does not need to spend time doing device power management if the
-devices are already put in the appropriate power state by the application or
-component managing the devices.
+Device runtime power management is a collaborative effort involving
+device drivers, subsystems, and applications. While device drivers
```

Comment:
not a fan of "collaborative effort". Would suggest something like the following: 

```suggestion
Device runtime power management involves coordinated interaction between
device drivers, subsystems, and applications. While device drivers
```

Reply:
this one I particularly prefer the original version, but I don't have hard opinion about it. Will change it.